### PR TITLE
 create task

### PR DESCRIPTION
**Pull request by brittajaison**

### CODE REVIEW

**Comment by sivabalaji1986** [comment]
@brittajaison
Please find the code review comments
* Please don't pass the access token in parameter; Access token in available in cookie. so the taskService can directly read it instead of Table or Dashboard javascript sending it

File: src/services/taskService.js [hunk 1]
```diff
@@ -78,4 +79,29 @@ const editTask = async(accessToken,taskId, taskData) => {
         throw error;
     }
 }
-export {fetchTaskList, deleteTask, editTask}
+const createTask = async (accessToken, taskData) => {
```
Comment (sivabalaji1986) [inline src/services/taskService.js:82]: Please read the accessToken from this method from cookie directly instead of passing it in all Javascripts

**Review by sivabalaji1986 — CHANGES_REQUESTED** [review]

File: src/services/taskService.js (hunk 1 above)
Comment (brittajaison) [reply src/services/taskService.js]: Fixed.

**Comment by brittajaison** [comment]
> @brittajaison Please find the code review comments
> 
> * Please don't pass the access token in parameter; Access token in available in cookie. so the taskService can directly read it instead of Table or Dashboard javascript sending it

Fixed.

File: src/services/taskService.js (hunk 1 above)
Comment (sivabalaji1986) [reply src/services/taskService.js]: Thanks. This looks fine...

**Pull request merged**